### ISSUE TITLE
Could this be applied in the tensorflow during the training process?

### ISSUE THREAD

**Issue by gaizixuan0128** [mattloper/opendr#22]
Hello!
I want to render a 3D model and get its silhouette using opendr, and then use this silhouette to calculate the cross entropy with the label mask image. I am wondering if it is differentialble during the training process......
Could this method work?

**Comment by nicolasugrinovic** [comment]
Any updates on this?

**Comment by zjmonk** [comment]
There is another project, https://github.com/google/tf_mesh_renderer

I haven't try it , but it is also a differentiable renderer that can be directly used in tensorflow.
